core/src/main/scripts/importer/validateData.py

```diff
@@ -288,26 +288,35 @@ def __init__(self, cancer_type_dict, hugo_entrez_map, alias_entrez_map, gene_set
                 for entrez_list in list(entrez_map.values()):
                     for entrez_id in entrez_list:
                         self.entrez_set.add(entrez_id)
-
-        # Set defaults for genome version and species
-        self.species = 'human'
-        self.ncbi_build = '37'
-        self.genome_build = 'hg19'
-
-    def load_genome_info(self, properties_filename):
-        """Retrieves the species and genome information from portal.properties."""
-        with open(properties_filename, 'r') as properties_file:
-            for line in properties_file:
-                line = line.strip()
-                if line.startswith('#') or '=' not in line:
-                    continue
-                sp_line = line.split('=', 1)
-                if sp_line[0] == 'species':
-                    self.species = sp_line[1]
-                elif sp_line[0] == 'ncbi.build':
-                    self.ncbi_build = sp_line[1]
-                elif sp_line[0] == 'ucsc.build':
-                    self.genome_build = sp_line[1]
+                        
+        #Set defaults for genome version and species
+        self.__species = 'human'
+        self.__ncbi_build = '37'
```

Comment:
should `37` be changed to `GRCh37` to be consistent with how we refer to ncbi builds in the java code? [see here](https://github.com/cBioPortal/cbioportal/pull/5891/files#diff-110497640cf5b4d5d5b73305afbd03caR72)